docs/pages/guides/monorepos.md

```diff
@@ -0,0 +1,153 @@
+
+- **apps/** - Contains multiple projects, including Expo apps.
+- **packages/** - Contains different packages used by our apps.
+- **package.json** - Root package file, containing yarn workspaces config.
+
+### Root package file
+
+All yarn monorepos should have a "root" **package.json** file. It is used as the main configuration for our monorepo and may contain packages installed for all projects in the repository. You can run `yarn init`, or create the **package.json** manually. It should look something like this:
+
+```JSON
+{
+  "name": "monorepo",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT"
+}
+```
+
+### Set up yarn workspaces
+
+Yarn and other tooling have a concept called _"workspaces"_. Every package and app in our repository has its own workspace. But, before we can use them, we have to instruct yarn where to find these workspaces. We can do that by setting the `workspaces` property using [glob patterns](https://classic.yarnpkg.com/lang/en/docs/workspaces/#toc-tips-tricks), in the **package.json**:
+
+```JSON
+{
+  "name": "monorepo",
+  "private": true,
+  "workspaces": [
+    "apps/*",
+    "packages/*"
+  ],
+  ...
+}
+```
+
+> ⚠️ Yarn workspaces require the root **package.json** to be private. If you don't set this, `yarn install` will error with a message mentioning this.
+
+### Create our first app
+
+Now that we have the basic monorepo structure set up let's add our first app. Before we can create our app, we have to create the **apps/** folder. This folder can contain all separate apps or websites that belong to this monorepo. Inside this **apps/** folder, we can create a subfolder that contains the actual Expo app. 
+
+```
+$ expo init apps/cool-app
+```
+
+> If you have an existing app, you can copy all those files inside a subfolder.
+
+After copying or creating the first app, run `yarn install` to make sure our workspace is working.
+
+### Create a package
+
+Monorepos can help us group code in a single repository. That includes apps but also separate packages. They also don't need to be published. The [Expo repository](https://github.com/expo/expo) uses this as well. In our repo, all the Expo SDK packages live inside the [**packages/**](https://github.com/expo/expo/tree/master/packages) folder. It helps us test the code inside one of our [**apps/**](https://github.com/expo/expo/tree/master/apps/native-component-list) before we publish them.
+
+Let's go back to the root and create the **package/** folder. This folder can contain all the separate packages that you want to make. Once you are inside this folder, we need to add a new subfolder. The subfolder is a separate package that we can use inside our app. In the example below, we named it **cool-package**.
+
+```
+$ mkdir -p packages/cool-package
```

Comment:
I kinda did this so I'd not have to explain everything in detail. It wasn't intended to be copied, because hey windows users. But I'll add that anyways.

Reply:
oh right, windows 👀

Reply:
Its funny how windows can't read this, but all windows developers can. Skynet still has a long way to go